### PR TITLE
New learning system graphshingling

### PR DESCRIPTION
**Pull request by idanivanov**
The source code of the system which I added can be found at: https://github.com/idanivanov/master_thesis.git

The local path to the tool needs to be configured in `learningsystems/graphshingling/graphshingling.conf.

The executable `run` builds feature vectors and saves them in a `temp` directory in the same location and runs SVM on a training set afterwards. `validate` runs SVM prediction on a validation set using the feature vectors from the execution of `run`. The split between training and validation data is hardcoded in both files. The seed for the random split can be changed in the configuration files for each learning problem.

Please, contact me for any questions.

### CODE REVIEW

**Comment by patrickwestphal** [comment]
Asked for feedback via DM after extending the graphshingling repo in an [own fork](https://github.com/patrickwestphal/master_thesis) but didn't get an answer.
